lib/DxcSupport/HLSLOptions.cpp

```diff
@@ -322,14 +322,26 @@ static bool handleVkShiftArgs(const InputArgList &args, OptSpecifier id,
   return true;
 }
 
+// Check if the given unsupported option is used with SPIR-V.
+static bool checkUnsupportedSpirvOption(const InputArgList &args,
```

Comment:
Maybe: `hasUnsupportedSpirvOption` or `containsUnsuportedSpirvOption`? The issue with `check` and `bool` as the return type is that it's not clear what the success/failure values are.

Reply:
Done